openqa-investigate

```diff
@@ -133,7 +133,14 @@ investigate() {
     # shellcheck disable=SC2181
     [[ $? != 0 ]] && echoerr "unable to query job data for $id: $job_data" && return 1
     old_name="$(echo "$job_data" | runjq -r '.job.test')" || return $?
-    [[ "$old_name" =~ ":investigate:" ]] && echo "Job is ':investigate:' already, skipping investigation" && return 0
+    if [[ "$old_name" =~ ":investigate:" ]]; then
+        job_result="$(echo "$job_data" | runjq -r '.job.result')" || return 0
+        [[ "$job_result" != "passed" ]] && echo "The investigate job $job_result, no need to comment back" && return 0
+        investigrate_origin="$(echo "$job_data" | runjq -r '.job.settings.OPENQA_INVESTIGATE_ORIGIN')" || return 1
+        origin_job_id=${investigrate_origin#$host_url/t}
+        "${client_call[@]}" -X POST jobs/"$origin_job_id"/comments text="Investigrate job: $host_url/t$id passed"
```

Comment:
It would make sense to state that this means the test likely is unstable:
```suggestion
        "${client_call[@]}" -X POST jobs/"$origin_job_id"/comments text="Investigrate job: $host_url/t$id passed, likely an unstable test"
```

Reply:
fixed. Thanks